src/test/java/uk/gov/hmcts/reform/sendletter/services/LetterServiceTest.java

```diff
@@ -250,4 +253,18 @@ public void should_not_allow_empty_service_name() {
         assertThatThrownBy(() -> service.send(letter, ""))
             .isInstanceOf(IllegalStateException.class);
     }
+
+    @Test
+    public void update_should_throw_an_exception_if_no_letters_were_updated() {
```

Comment:
It would be good to have another one, checking if the service feeds the repository with the right data.

Reply:
added